app/Http/Controllers/PagesController.php

```diff
@@ -492,49 +492,21 @@ public function graphs()
 		}
 
 		// Analysis of participants' preference for camp type (from reviews)
-		$camp_prefs = [];
-		$revs = \App\Review::whereNotNull('kampkeuze')->pluck('kampkeuze')->toArray();
-		$prefs_rev_count = count($revs);
-		foreach ($revs as $opts) {
-			foreach ($opts as $opt) {
-				if (array_key_exists($opt, $camp_prefs)) {
-					$camp_prefs[$opt]++;
-				} else {
-					$camp_prefs[$opt] = 1;
-				}
-			}
+		$reviews = \App\Review::whereNotNull('kampkeuze')->pluck('kampkeuze')->toArray(); // note: 2D array
+
+		$reviews_flat = [];
+		foreach ($reviews as $r) {
+			$reviews_flat = array_merge($reviews_flat, $r);
```

Comment:
```suggestion
		$reviews = \App\Review::whereNotNull('kampkeuze')->pluck('kampkeuze');
		$prefs_review_count = $reviews->count();
		$reviews_flat = $reviews->flatten()->toArray();
```
This can be simplified to this above.  The `$prefs_review_count` is also done with laravel collections this way.
I'd like to fallback to laravel collections as much as possible, as it is easier to read, and more consistent with laravel itself.

Reply:
I've actually been searching for a php way to flatten an array but couldn't find any. Didn't think to check the Laravel specific methods... nice one.